include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -228,6 +230,23 @@ class HostManager final {
   /// Get the network DAG for \p network if it exists.
   Expected<DAG *> getNetworkDAG(llvm::StringRef network);
 
+  /// \returns a non-owning pointer to the TraceContext.
+  TraceContext *getTraceContext() { return hostTraceContext_.get(); }
+
+  /// Sets the TraceContext and \returns the existing value.
+  std::unique_ptr<TraceContext>
+  setTraceContext(std::unique_ptr<TraceContext> traceContext) {
+    std::swap(hostTraceContext_, traceContext);
+    return traceContext;
+  }
+
+  /// Triggers start tracing of all active devices \returns Error if fails.
```

Comment:
The comment here says this will return an error if it fails, but it's returning void. I guess we should make it return a glow error or something.